### PR TITLE
Creating configs for Whitespace (Non-Comp) Package

### PR DESCRIPTION
**Pull request by relentless-pursuit**
Part of #52

### CODE REVIEW

File: WhitespaceAfter/all-examples-in-one/config-all-in-one.xml [hunk 1]
```diff
@@ -20,7 +20,6 @@
     <module name="TreeWalker">
         <module name="WhitespaceAfter">
             <property name="id" value="example1"/>
-            <property name="tokens" value="COMMA, SEMI"/>
```
Comment (romani) [inline WhitespaceAfter/all-examples-in-one/config-all-in-one.xml:23]: https://checkstyle.org/checks/whitespace/whitespaceafter.html#WhitespaceAfter

there are 2 examples in this Check
Why we missing example2 ?

**Review by romani — CHANGES_REQUESTED** [review]
items:

File: WhitespaceAfter/all-examples-in-one/config-all-in-one.xml (hunk 1 above)
Comment (relentless-pursuit) [reply WhitespaceAfter/all-examples-in-one/config-all-in-one.xml:23]: > https://checkstyle.org/checks/whitespace/whitespaceafter.html#WhitespaceAfter
> 
> there are 2 examples in this Check Why we missing example2 ?

this is from non-compilable source: https://github.com/checkstyle/checkstyle/tree/master/src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace

Comment (relentless-pursuit) [reply WhitespaceAfter/all-examples-in-one/config-all-in-one.xml:23]: image

Comment (romani) [reply WhitespaceAfter/all-examples-in-one/config-all-in-one.xml:23]: We need to take example1 from https://github.com/checkstyle/checkstyle/tree/master/src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter

**Comment by romani** [comment]
This PR looks like blocked by https://github.com/checkstyle/test-configs/issues/81

File: WhitespaceAfter/all-examples-in-one/config-all-in-one.xml (hunk 1 above)
Comment (relentless-pursuit) [reply WhitespaceAfter/all-examples-in-one/config-all-in-one.xml:23]: > We need to take example1 from https://github.com/checkstyle/checkstyle/tree/master/src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter

Yes, this PR is taking Example1 from this path.

**Comment by romani** [comment]
@relentless-pursuit , please continue this PR.

**Comment by relentless-pursuit** [comment]
> @relentless-pursuit , please continue this PR, referenced issue is closed.

I have already covered the changes of this PR in https://github.com/checkstyle/test-configs/pull/92

**Pull request closed without merging**